.github/workflows/publish.yml

```diff
@@ -37,3 +37,95 @@ jobs:
         run: npm publish --tag ${{ steps.npm_tag.outputs.npm_tag }}
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+      - name: Wait for 300 seconds
+        run: sleep 300s
+        shell: bash
+  deploy_chime_prod_demo:
+    needs: publish
+    name: Prod - Chime Client - Deploy the Serverless Meeting Demos
+    runs-on: ubuntu-latest
+    env:
+      AWS_DEFAULT_REGION: us-east-1
+      AWS_DEFAULT_OUTPUT: text
+      NAME: ChimeProd
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.PROD_AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.PROD_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.release.tag_name }}
+      - name: Get current published version
+        id: npm_version
+        run: |
+          current_version=$(.github/script/get-current-version)
+          echo "Current released version:" $current_version
+          echo ::set-output name=npm_version::$current_version
+      - name: Install SDK and verify npm version
+        run: |
+          cd demos/browser
+          npm uninstall amazon-chime-sdk-js
+          npm install amazon-chime-sdk-js@${{ steps.npm_version.outputs.npm_version }}
+      - name: Verify npm version
+        run: |
+          demo_current_version=$(.github/script/get-demo-browser-current-version)
+          echo "Current demo version:" $demo_current_version
+          if [[ *$demo_current_version* != "^#${{ steps.npm_version.outputs.npm_version }}" ]];
```

Comment:
Not sure the value of this check here. I dont think i ever see the installation failed to install the correct version. If you want to check for it, better to exit 1 to mark the job to fail. Otherwise, noone would know.

Reply:
It was in the old script, so added here too, I am not sure why this was added. If this is needed, will add Exit 1